spec/lib/pubmed/harvester_spec.rb

```diff
@@ -33,8 +35,26 @@
 
     context 'when author query has too many publications' do
       let(:lotsa_pmids) { Array(1..Settings.PUBMED.max_publications_per_author) }
+      let(:query_author) { instance_double(Pubmed::QueryAuthor) }
+
+      before do
+        allow(Pubmed::QueryAuthor).to receive(:new).with(author, {}).and_return(query_author)
+        allow(query_author).to receive(:pmids).and_return(lotsa_pmids)
```

Comment:
You can incorporate these lines into the instance_double call

Reply:
done